### PR TITLE
add ssr custom directive test

### PR DESCRIPTION
**Pull request by kazupon**
ref: #2872

### CODE REVIEW

File: test/ssr/ssr-string.spec.js [hunk 1]
```diff
@@ -360,6 +358,39 @@ describe('SSR: renderToString', () => {
     })
   })
 
+  it('custom directives', done => {
+    const custom = jasmine.createSpy()
```
Comment (yyx990803) [inline test/ssr/ssr-string.spec.js]: We should implement some mock behavior of manipulating the VNode and assert the result. (see https://github.com/vuejs/vue/tree/next/src/platforms/web/server/directives)

Comment (kazupon) [inline test/ssr/ssr-string.spec.js]: thanks!
I'll try to fix it. :)

Comment (kazupon) [inline test/ssr/ssr-string.spec.js]: Done

**Comment by yyx990803** [comment]
👍

**Pull request merged**